features/FxA-49-cdn/README.md

```diff
@@ -0,0 +1,91 @@
+
+# CDN
+
+https://mozilla.aha.io/features/FXA-49
+
+## Stories
+
+As a user, I want Firefox Accounts UI to load fast
+so that it feels responsive and I can get on with
+that I'm doing.
+
+## Details
+
+Firefox Accounts serves all content from datacenters located
+on the west-coast of the US. For users located in
+geographically distant regions of the globe, the time it
+takes to download the static resources is a major contributor
+to overall application start time. By pushing static resources
+closer to the user, the hope is users experience a faster
+percieved start time.
+
+All static resources will be served from <XXX>.
+
+### Measuring Success
+
+* `navigationTiming` data - for DOMContentLoaded
+* `resourceTiming` data of individual resources.
+  The largest of the static assets is `main.js`
+* `loaded` event in metrics - when the user sees the
+  first screen - the user based metric that matters.
+
+It is expected that both `DOMContentLoaded` from navigationTiming
+data and the `loaded` message in internal metrics will decrease,
+especially for users located in geographically distant regions
```

Comment:
If we're going to look at aggregate metrics, let's be more explicit about exactly what and how.  e.g. will we look at the mean, median, 95th percentile of these measures?  This will help us make sure we've got the appropriate graphs in place.  I think it'll be worth monitoring these on an ongoing basis.